aws/error.go

```diff
@@ -0,0 +1,7 @@
+package aws
+
+import "github.com/giantswarm/microerror"
+
+var invalidConfigError = microerror.Newf("invalid config error")
+
+var executionFailedError = microerror.Newf("execution failed error")
```

Comment:
Let's replace that with the code below. It won't work with future microerror releases as `New` and `Newf` are removed.

```go
var executionFailedError = &microerror.Error{
	Kind: "executionFailedError",
}

var invalidConfigError = &microerror.Error{
	Kind: "invalidConfigError",
}

// IsInvalidConfig asserts invalidConfigError.
func IsInvalidConfig(err error) bool {
	return microerror.Cause(err) == invalidConfigError
}
```

Reply:
thx